src/shared/Map/NitrogenMap.jsx

```diff
@@ -31,7 +32,10 @@ const NitrogenMapComp = ({ variant }) => {
   const [features, setFeatures] = useState(mapPolygon);
   const [drawEvent, setDrawEvent] = useState({});
 
+  const mapboxToken = import.meta.env.VITE_MAPBOX_TOKEN;
```

Comment:
can you make a keys.js file similar to the selector and then import the env files from there to be consisten with other tools?